exasol/python_extension_common/deployment/language_container_deployer.py

```diff
@@ -87,6 +87,10 @@ def get_udf_path(bucket_base_path: bfs.path.PathLike, bucket_file: str) -> PureP
     return PurePosixPath(file_path.as_udf_path())
 
 
+def display_extract_progress(n: int, pending: List[int]):
```

Comment:
How often would this be called?

Reply:
It depends on the settings. 

Default is extract-timeout-minutes (currently 10 minutes, but under discussion) 
and the interval (currently, always set to 10 secs, not configurable via CLI). 

So with these setting, by default the callback will be called 10*60/10 = 60 times at max.

Reply:
I probably, would increase the interval to 30 seconds

Reply:
Thanks - changes in latest push.